Examples/Tests/reduced_diags/analysis_reduced_diags_impl.py

```diff
@@ -306,4 +306,6 @@ def do_analysis(single_precision = False):
         print()
 
     test_name = os.path.split(os.getcwd())[1]
-    checksumAPI.evaluate_checksum(test_name, fn)
+
+    checksum_rtol = 2e-9 if single_precision else 1e-9
```

Comment:
Going by some other tests, this could justifiably be **much** looser, but that's not necessary for these to pass, so I don't stretch things.